Add swagg recipe

isamert opened the pull request:
### Brief summary of what the package does

Basic Swagger UI for swagger.json/yaml files, based on `completing-read`.

### Direct link to the package repository

https://github.com/isamert/swagg.el

### Your association with the package

I am the maintainer

### Relevant communications with the upstream package maintainer

**None needed**

### Checklist

<!-- Please confirm by replacing `[]` with `[x]`: -->

- [x] The package is released under a [GPL-Compatible Free Software License](https://www.gnu.org/licenses/license-list.en.html#GPLCompatibleLicenses)
- [x] I've read [CONTRIBUTING.org](https://github.com/melpa/melpa/blob/master/CONTRIBUTING.org)
- [x] I've used the latest version of [package-lint](https://github.com/purcell/package-lint) to check for packaging issues, and addressed its feedback
- [x] My elisp byte-compiles cleanly
- [x] I've used `M-x checkdoc` to check the package's documentation strings
- [x] I've built and installed the package using the instructions in [CONTRIBUTING.org](https://github.com/melpa/melpa/blob/master/CONTRIBUTING.org)

<!-- After submitting, please fix any problems the CI reports. -->

riscy commented:
Thanks for this.  Quick first pass --

`swagg.el` with byte-compile using Emacs 28.1:
```
swagg.el:549:4: Warning: the function `pp-display-expression' is not known to
    be defined.
```
Maybe this is defined in Emacs 29.1?

`swagg.el` with package-lint 20230806.1331:
```
3 issues found:
9:92: error: Package json is not installable.
529:40: error: You should depend on (emacs "29.1") if you need `json-ts-mode'.
705:11: error: You should depend on (emacs "29.1") if you need `with-memoization'.
```
(Putting `json` in your `Package-Requires` isn't necessary)

isamert commented:
> (Putting `json` in your `Package-Requires` isn't necessary)

Removed this.

> ```
> swagg.el:549:4: Warning: the function `pp-display-expression' is not known to
>   be defined.
> ```

Interestingly `package-lint` did not warn me about this. I added `(require 'pp)` which should fix it I guess.

> ```
> 529:40: error: You should depend on (emacs "29.1") if you need `json-ts-mode'.
> ```

I removed `json-ts-mode` usage, it was not really necessary anyway.

> ```
> 705:11: error: You should depend on (emacs "29.1") if you need `with-memoization'.
> ```

I am using [compat.el](https://git.sr.ht/~pkal/compat) for this to work in older versions of Emacs but seems like [package-lint does not support it](https://github.com/purcell/package-lint/issues/227) right now.

riscy commented:
> Interestingly package-lint did not warn me about this.

This was a byte-compile one, but it looks good now.

> I am using [compat.el](https://git.sr.ht/~pkal/compat) for this to work

I didn't notice, and that's on me. :)  Thank you for accommodating these.
